### PR TITLE
fix wrong reference to package

### PR DESCRIPTION
**Pull request by dotob**
need different format for npm

### CODE REVIEW

**Comment by dotob** [comment]
hmm do you think this has todo with my changes? i think the package resolve does work again now.

File: package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "homepage": "https://github.com/peerlibrary/node-xml4js",
   "version": "0.1.0",
   "dependencies": {
-    "xml2js": "https://github.com/peerlibrary/node-xml2js.git#peerlibrary",
+    "xml2js": "git+https://git@github.com/peerlibrary/node-xml2js.git",
```
Comment (mitar) [inline package.json]: You lost the branch. This is why it is failing.

Comment (mitar) [inline package.json]: I don't think you need `git@` with https.

**Comment by mitar** [comment]
Yes. Thanks again.

**Pull request merged**